src/ref_count.rs

```diff
@@ -77,8 +93,10 @@ fn total_chunks(index_bits: u8) -> u64 {
 	1u64 << index_bits
 }
 
-fn file_size(index_bits: u8) -> u64 {
-	total_entries(index_bits) * ENTRY_BYTES as u64 + META_SIZE as u64
+fn file_size(index_bits: u8, max_chunks: Option<u64>) -> u64 {
+	let max_size = max_chunks.map(|c| c * CHUNK_LEN as u64);
+	let total = total_entries(index_bits) * 8 + META_SIZE as u64;
```

Comment:
Should be:
`let total = total_entries(index_bits) * ENTRY_BYTES as u64 + META_SIZE as u64;`
Ref count entries are 16 bytes, not 8.

Reply:
Indeed, awfull copy paste here.

Reply:
```suggestion
	let total = total_entries(index_bits) * ENTRY_BYTES as u64 + META_SIZE as u64;
```